sunkit_image/flct/flct.py

```diff
@@ -0,0 +1,499 @@
+          format.
+        * If you use the IDL IO routines to get the input arrays from ``dat`` files,
+          the IDL routines always read the binary files in the column major, but both Python and C,
+          on which these functions are written are row major so the order of the arrays have to be
+          changed which can be done with the ``order`` keyword. This may lead to different values
+          in both the cases.
+        * If your input arrays are column major then pass the `order` parameter as `column` and it
+          will automatically take care of the order change. But this can produce some changes in
+          the values of the arrays.
+
+    Parameters
+    ----------
+    image1 : `numpy.ndarray`
+        The first image of the sequence of two images on which the procedure is to be performed.
+    image2 : `numpy.ndarray`
+        The second image of the sequence of two images taken after ``deltat`` time of the first one.
+    order : {"row" | "column"}
+        The order in which the array elements are stored that is whether they are stored as row
+        major or column major.
+    deltat : `float`
+        The time interval between the two images.
+    deltas : `float`
+        Units of length of the side of a single pixel. Velocity is computed in units of deltas/deltat.
+    sigma : `float`
+        The width of Gaussian kernel with which the images are to be modulated. Sub-images are weighted
+        by Gaussian of width sigma. If sigma is ``0`` then the overall shift between the images is
+        returned.
+    quiet : `bool`
+        If set to `True` all the error messages of FLCT C code will be suppressed.
+        Defaults to `False`.
+    biascor : `bool`
+        If set to `True` bias correction will be applied while computing the velocities.
+        This bias is intrinsic to the FLCT algorithm. Due to this it underestimates the velocities
+        during calculations.  The bias correction algorithm that is implemented uses the Hessian
+        determinant, and the assumed value of sigma, to adjust the x-and-y velocities. For more
+        details visit `here <http://solarmuri.ssl.berkeley.edu/~fisher/public/software/FLCT/C_VERSIONS/flct_1.06/doc/bias_correction_in_flct.txt>`__.
+    thresh : `float`
+        The FLCT calculations will not be done for a particular pixel if the average absolute value
+        between the two images is less than `thresh` at that particular pixel. The average absolute
+        value is calculated by taking the average of absolute value of intensity in both the images
+        at that particular pixel. If ``thresh`` is between 0 and 1, ``thresh`` is assumed given in
+        units relative to the largest absolute value of the image average.
+        Defaults to 0.
+    absflag : `bool`
+        This is set to `True` to force the ``thresh`` values between 0 and 1 to be considered in
+        absolute terms.
```

Comment:
Does this not need to be set to True if a threshold of 0-1 is given?

Reply:
No I don't think so. For 0-1 also it is upto the user to decide.
```
printf("-t thr - if avg abs value image pixel < thr, skip calc\n");
      printf("   (if thr between 0 & 1, thr is assumed to be in\n");
      printf("   relative units of the max. abs. pixel value.  To force\n");
      printf("   thr to be in absolute units, append an 'a' to it.)\n");
```

Reply:
But doesn't the docstring above say that it was taken to be that?

Reply:
Yes it says it was taken to  be realtive and we want it to be relative by default. The default way `thresh` values between (0,1) are taken is relative but if the user wants them as absolute then he has to specify the absflag.
Doesn't it make sense that the thresh value is 0.5 of maximum (suppose) (this is relative) rather than being 0.5 (a number) (this is absolute)
I am not sure whether I understand your concern over this? Am I missing something?

Reply:
No its fine. I re-read the doc string, I misunderstood it.